### ISSUE TITLE
Awards duplicated

### ISSUE THREAD

**Issue by brendanarciszewski** [JustDerek1/FRCCentralIssues#18]
781 is listed as having won 2 Buckeye Regionals in 2016.
![img_0540](https://cloud.githubusercontent.com/assets/20262406/20505197/5b508946-b019-11e6-9b95-b5a85c435b24.JPG)

**Comment by brendanarciszewski** [comment]
Also, the Victoria Park event did not exist last year, that award should be from Greater Toronto Central, although this seems to come from part of #17

**Comment by JustDerek1** [comment]
Yah, I saw this earlier.  Pain in the butt.  I think it is a data issue.  I think I have duplicate items loaded.

**Comment by JustDerek1** [comment]
Yep, it was a data issue.  Reloaded data.  So, you will have to delete the app and redownload it to have it work.  It caches the awards data.  Technically awards data shouldn't change, so it currently doesn't refresh it.  (That's an improvement that I hope to make before release.)

**Comment by JustDerek1** [comment]
I'm going to reopen this.  I encountered this again over the weekend when I was playing around.
